docs/quests.proto

```diff
@@ -102,6 +101,22 @@ message UserUpdate {
   }
 }
 
+message Quests {
+  repeated QuestInstance quests = 1;
+}
+
+message ProtoQuest {
```

Comment:
why not `Quest`?

Reply:
It collides with the Rust struct definition: https://github.com/decentraland/quests/blob/main/crates/protocol/src/quests.rs#L13